packages/orbit-components/src/Box/Box.stories.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from "react";
-import { text, select, number } from "@storybook/addon-knobs";
 
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 import type { SpacingToken, ColorTokens } from "./types";
 
 import Box from ".";
 
+Box.displayName = "Box";
```

Comment:
I added explicit displayName because `react-element-to-jsx-string` has [issue with components using forwardRef](https://github.com/algolia/react-element-to-jsx-string/issues/859), it displays "No" instead of component name. It started from SB v8 and setting displayName solves it.